test/hotspot/jtreg/compiler/codecache/stress/OverloadCompileQueueTest.java

```diff
@@ -52,8 +53,33 @@
 import java.util.stream.IntStream;
 import java.util.Random;
 
-public class OverloadCompileQueueTest implements Runnable {
+class LockUnlockThread extends Thread {
     private static final int MAX_SLEEP = 10000;
+    private static final int DELAY_BETWEEN_LOCKS = 100;
+    private final Random rng = Utils.getRandomInstance();
+
+    public volatile boolean isActive = true;
+
+    @Override
+    public void run() {
+        try {
+            while (isActive) {
+                int timeInLockedState = rng.nextInt(MAX_SLEEP);
+                Helper.WHITE_BOX.lockCompilation();
+                Thread.sleep(timeInLockedState);
+                Helper.WHITE_BOX.unlockCompilation();
+                Thread.sleep(DELAY_BETWEEN_LOCKS);
+            }
+        } catch (InterruptedException e) {
+            throw new Error("TESTBUG: lockUnlocker thread was unexpectedly interrupted", e);
```

Comment:
Since `lockUnlocker` method is gone, the message should be "LockUnlockThread was unexpectedly interrupted"?

Also, throwing the error from this thread would not be rethrown with `join` later. If we care about this error, should we instead store it into `static final` field here, check it after `join`, and rethrow if ` != null`. The old code seems to have the same problem, though, so we can keep ignoring it.

Reply:
Fixed the message. Regarding the error - it is an Error, not a Throwable, it gets noticed and recorded by the jtreg (I checked that).